src/pages/dashboard/staking-pools/SafetyPoolDetail.tsx

```diff
@@ -153,40 +281,135 @@ const SafetyPoolDetail: React.FC<
                       params: { SYMBOL: AssetSymbol.DYDX },
                     })
               }
-              // ctaConfig={{
-              //   color: ButtonColor.Lighter,
-              //   disabled: !walletAddress,
-              //   label: stringGetter({ key: STRING_KEYS.WITHDRAW }),
-              //   onClick: () => openModal({ type: ModalType.Claim }),
-              // }}
-              isLoading={!!walletAddress && !safetyPoolStakingBalances.userBalance}
-            />
-            <SingleStatCard
-              color={walletAddress ? CardColor.Light : CardColor.Dark}
-              title={stringGetter({ key: STRING_KEYS.EARNED })}
-              value={earnings}
-              label={stringGetter({
-                key: userHasStakingBalance
-                  ? STRING_KEYS.UNCLAIMED_REWARDS
-                  : STRING_KEYS.STAKE_TO_EARN_REWARDS,
-              })}
-              ctaConfig={{
-                disabled: true,
-                label: stringGetter({ key: STRING_KEYS.CLAIM }),
-                onClick: () => openModal({ type: ModalType.Claim }),
-              }}
-              isLoading={!!walletAddress && !safetyPoolStakingBalances.unclaimedRewards}
+              isLoading={!!walletAddress && (!userBalance || !availableWithdrawBalance)}
             />
+            {!isUserGeoBlocked && (
+              <SingleStatCard
+                color={walletAddress ? CardColor.Light : CardColor.Dark}
+                title={stringGetter({ key: STRING_KEYS.EARNED })}
+                value={formattedEarnings}
+                label={stringGetter({
+                  key: userHasStakingBalance
+                    ? STRING_KEYS.UNCLAIMED_REWARDS
+                    : STRING_KEYS.STAKE_TO_EARN_REWARDS,
+                })}
+                ctaConfig={
+                  MustBigNumber(unclaimedRewards).gt(0)
+                    ? {
+                        disabled: !walletAddress,
+                        label: stringGetter({ key: STRING_KEYS.CLAIM }),
+                        onClick: () => openModal({ type: ModalType.Claim }),
+                      }
+                    : undefined
+                }
+                isLoading={!!walletAddress && !unclaimedRewards}
+              />
+            )}
           </CardRow>
-        </ContentLeft>
-        <ContentRight>
+          {walletAddress && (
+            <Styled.WithdrawSection>
```

Comment:
This could possible be moved into the above `if (walletAddress)`

Reply:
I think that might be a little too much, let's keep it the way it is